vtkext/private/module/Testing/TestF3DEXRMemReader.cxx

```diff
@@ -0,0 +1,57 @@
+#include <vtkImageData.h>
+#include <vtkNew.h>
+
+#include "vtkF3DEXRReader.h"
+
+#include <fstream>
+#include <iostream>
+#include <vector>
+/**
+ * image file taken from
+ * https://github.com/AcademySoftwareFoundation/openexr/blob/370db2835843ac75f85e1386c05455f26a6ff58c/website/test_images/Chromaticities/Rec709.rst
+ */
+bool readFileToVector(const std::string& filename, std::vector<char>& buffer)
+{
+  std::ifstream file(filename, std::ios::binary | std::ios::ate);
+  if (!file)
+  {
+    return false;
+  }
+
+  std::streamsize size = file.tellg();
+  file.seekg(0, std::ios::beg);
+
+  buffer.resize(size);
+
+  if (file.read(buffer.data(), size))
+    return true;
+  return false;
```

Comment:
I tried this but the return from file.read() is istream& and cannot be converted to a bool so i think this is the cleanest way to return.

Reply:
I thought `operator bool` (https://en.cppreference.com/w/cpp/io/basic_ios/operator_bool) would have been called automatically.  
If not, then you're implementation is fine, feel free to ignore my comment.

Reply:
can it be cast explicitly? something like `return (bool) file.read(buffer.data(), size);`?
Otherwise `return file.read(buffer.data(), size)? true : false;` to keep it one line instead of 3 (which will be 5 with the Allman style)?